sdf_timing/sdflex.py

```diff
@@ -122,12 +123,17 @@
 
 
 # define FLOAT as function so it takes precendence over STRING
+def t_FIXPOINT(t):
+    r"[0-9]*'[bohd][0-9]+"
```

Comment:
hmm according to the sdf spec only binary constants are supprted (see http://ce.sharif.edu/courses/85-86/2/ce483/resources/root/sdf_3.0.pdf page 83:
```
SCALAR_CONSTANT ::= 1‘b0 // logical zero
                ||= 1‘b1 // logical one
                ||= 1‘B0 // logical zero
                ||= 1‘B1 // logical one
                ||= ‘b0 // logical zero
                ||= ‘b1 // logical one
                ||= ‘B0 // logical zero
                ||= ‘B1 // logical one
                ||= 0 // logical zero
                ||= 1 // logical one
```

Reply:
Renamed FIXPOINT into SCALARCONSTANT to better reflect this. I have also tightened the regex to better reflect that only binary constants are allowed. 

The commit ab3ba07 below passes the tests. I haven't worked with ply before, so on a more fundamental level, I am unsure if this is the "best" way to solve this. Technically, FLOAT are not allowed at this point in the as far as I can tell, thus we should get rid of it in "equation", then the SCALARCONSTANT has to match on 0 and 1 as well. However, this results in ambiguous grammar, which makes some of the tests fail. So instead, I left it as is.